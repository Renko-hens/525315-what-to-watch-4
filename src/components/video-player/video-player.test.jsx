import React, {createRef} from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

describe(`Render VideoPlayer Component`, () => {
  it(`Redner VideoPlayer`, () => {
    const tree = renderer.create(
        <VideoPlayer videoRefer={createRef()}/>, {
          createNodeMock: () => {
            return {};
          }}).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
