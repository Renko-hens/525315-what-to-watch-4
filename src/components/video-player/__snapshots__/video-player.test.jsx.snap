// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayer Component Redner VideoPlayer 1`] = `
<video
  autoPlay={true}
  className="player__video"
/>
`;
